CoreStubs/ArchiveControl/module.php

```diff
@@ -54,9 +74,26 @@ public function DeleteVariableData(int $VariableID, int $StartTime, int $EndTime
         throw new Exception('Not implemented');
     }
 
-    public function GetAggregatedValues(int $VariableID, int $AggregationLevel, int $StartTime, $EndTime, $Limit)
+    public function GetAggregatedValues(int $VariableID, int $AggregationLevel, int $StartTime, int $EndTime, int $Limit)
     {
-        throw new Exception('Not implemented');
+        if ($this->AggregatedArchive == []) {
```

Comment:
Finde ich gut von @DrNiels. Sollten wir bzgl. Konsistenz direkt in GetLoggedValues ebenfalls einbauen. Ich mag übrigens "empty()". Bin da flexibel :)